tests/suite/resources_utils.py

```diff
@@ -385,12 +385,12 @@ def ensure_item_removal(get_item, *args, **kwargs) -> None:
     """
     try:
         counter = 0
-        while counter < 20:
+        while counter < 30:
```

Comment:
Do you think it makes sense to make this as a variable/parameter? eg if this is going to be changed many times because as tests get slower/more complex?

Reply:
This shouldn't really depend on a complexity of the tests. I plan to keep an eye on this and if we need to increase this again then we'll have to investigate the cause deeper.

Reply:
Ok thanks!